app/src/main/res/xml/preferences.xml

```diff
@@ -109,6 +109,14 @@
             android:key="prefHardwareForwardButtonSkips"
             android:summary="@string/pref_hardwareForwardButtonSkips_sum"
             android:title="@string/pref_hardwareForwardButtonSkips_title"/>
+        <Preference
```

Comment:
I didn't implement this preference for prefHardwareForwardButtonSkips.  I fixed a typo in the name that already existed.